src/taoensso/sente.cljx

```diff
@@ -928,8 +943,10 @@
               false))))))
 
   (-chsk-connect! [chsk]
-    (when-let [WebSocket (or (enc/oget js/window "WebSocket")
-                             (enc/oget js/window "MozWebSocket"))]
+    (when-let [WebSocket (if is-node?
+                           node-websocket
```

Comment:
Can we include `node-websocket` in the `or` conditional below? It'll be nil if not running on node.